### PR TITLE
New Districts

### PR DESCRIPTION
**Pull request by magicksRampage**
-Updated the flavor text for every single district in the game
-Updated the property class values of many districts in the game
-Added three new districts in the southwest of the map
-Added 5 (technically 6) new sub-zones
-Added 2 new vendors
-Added the Ferry
-Disconnected Vandal Park from Cop Killtown

### CODE REVIEW

File: ewcfg.py [hunk 1]
```diff
@@ -2413,7 +2430,7 @@ def wef_scythe(ctn = None):
 		str_in = "at the base of",
 		str_enter = "arrive at",
 		str_name = "ENDLESS WAR",
-		str_desc = "ENDLESS WAR towers above you, its gray color almost making it seem to disappear against the gray of the sky. Its eye stares longingly into the cityscape.",
+		str_desc = "It’s bright, neon green color nearly blinds you when observed from this close. You are overwhelmed by an acute, menacing aura as you crane your neck to observe the obelisk its entirety. You almost thought you saw it looking back down at you, but it was probably just your imagination. You shouldn’t stay here any longer than you have to, you always get a weird feeling in the pit of your stomach when you stick around for too long.",
```
Comment (krakissi) [inline ewcfg.py]: "It’s bright, neon green..." should be "Its"

"obelisk its entirety" should be "obelisk in its entirety"

File: ewcfg.py [hunk 2]
```diff
@@ -2425,8 +2442,9 @@ def wef_scythe(ctn = None):
...
+		str_desc = "Here, businessmen carrying briefcases dripping with slime powerwalk from every direction to every other direction. They barely acknowledge your existence outside of muttering under their breath when they’re forced to sidestep around you and the other clueless juveniles loitering in their lobby. Above the first few floors begins the endless labyrinths of cubicles and office spaces that comprise the majority of the building. This corporate nightmare repeats itself for nearly every floor of the towering skyscraper. With it’s sleek, modern architecture and high-tech amenities, SlimeCorp HQ looks nothing like the rest of the city.\nPast countless receptionists' desks, waiting rooms, legal waivers, and at least one or two stainless steel vault doors, lay several slime donation rooms. All that wait for you in these secluded rooms is a reclined medical chair with an attached IV bag and the blinding light of a fluorescent light bulb. If you choose to !donate some of your slime, a SlimeCorp employee will take you to one of these rooms and inform you of the vast and varied uses of SlimeCoin, SlimeCorps’ hot new cryptocurrency.",
```
Comment (krakissi) [inline ewcfg.py]: " With it’s sleek, modern " should be "its"

"SlimeCorps’ hot new" should be "SlimeCorp's"

File: ewcfg.py [hunk 3]
```diff
@@ -2441,7 +2459,7 @@ def wef_scythe(ctn = None):
...
...nd. Hordes of men and women from every imaginable background walk these cruel streets, trying desperately to eke out a pitiful existence for themselves. This district never unwinds from it’s constant 24/7 slime-induced mania for even a moment, let alone sleep.\nDowntown is the beating heart of New Los Angeles City, aka Neo Milwaukee. With settlements in the area predating the emergence of slime, it’s prime location along the newly formed coastline naturally grew it into the cultural, economic, and literal center of the city. Due to it’s symbolic and strategic importance, it is home to the most intense gang violence of the city. Gunshots and screams followed by police sirens are background noises for this district. Some say that this propensity for violence is result of the sinister influence from an old obelisk in the center of town, ominously called ENDLESS WAR. You aren’t sure if you believe that, though.\n\nThis area contains ENDLESS WAR, SlimeCorp HQ and the Slime Stock Exchange. To the north is Smogsburg. To the East is the Green Light District. To the South is the Rowdy Roughhouse. To the Southwest is Poudrin Alley. To the West is Krak Bay. To the Northwest is Cop Killtown.",
```
Comment (krakissi) [inline ewcfg.py]: " from it’s constant " ... "its" (multiple)

File: ewcfg.py [hunk 4]
```diff
@@ -2462,11 +2480,11 @@ def wef_scythe(ctn = None):
...
+		str_desc = "In every direction, smokestacks belch out copious amounts of pollution into the atmosphere, creating a thick cloud that shrouds the district in sickening smog. It covers the district so completely that you can barely make out what time day it is. Your lungs can’t take much more of standing here, just do what you want to do and get out.\nSmogsburg is comprise of dozens of slime refineries and poudrin mills that turn unrefined, raw materials like the sludge from the city’s harbor into useful, pure slime. Functioning as the city’s premier industrial sector, it is by far the district hardest on the environment.\n\nThis area contains the Bazaar. To the North is Arsonbrook. To the Northeast is Little Chernobyl. To the East is Old New Yonkers. To the South is Downtown NLACakaNM. To the West is Cop Killtown. To the Northwest is Astatine Heights.",
```
Comment (krakissi) [inline ewcfg.py]: "is comprise of dozens" ... "comprised"

File: ewcfg.py [hunk 5]
```diff
@@ -2501,7 +2519,7 @@ def wef_scythe(ctn = None):
...
+		str_desc = "Long street blocks are densely packed stores and restaurants, mixed in with townhouses and accompanied by modern skyscrapers and sprawling in-door shopping malls. These amenities and a scenic view of the River of Slime on it’s coast makes this district a favorite of a juvenile out on the town.\nKrak Bay is a bustling commercial district, featuring stores from across the retail spectrum. From economic, practical convenience stores to high-class, swanky restaurants, Krak Bay has it all. It is also home to some of the most recognizable fixtures of the city’s skyline, most notably the Poudrintial Tower and the shopping mall at its base which contains the city’s prized food court.\n\nThis area contains the Food Court. To the East is Downtown NLACakaNM. To the Southeast is Poudrin Alley. To the South is Ooze Gardens. To the Southwest is South Sleezeborough. To the West is North Sleezeborough. To the Northwest is Glocksbury.",
```
Comment (magicksRampage) [inline ewcfg.py]: "are densely packed stores and restaurants" should be "are densely packed with stores and restaurants"

"on it's coast" should be "on its coast"

File: ewcfg.py [hunk 6]
```diff
@@ -2564,7 +2582,7 @@ def wef_scythe(ctn = None):
 			"ony"
 		],
 		str_name = "Old New Yonkers",
-		str_desc = "Rows of three-story brick and stone condominiums with white marble moulding wind along lanes of chipped cobblestone. Wrought-iron spikes jut from windows and balconies.\n\nTo the Northeast is New New Yonkers. To the Southeeast is Vagrant's Corner. To the Southwest is Smogsburg. To the East is Little Chernobyl. To the Northwest is Brawlden.",
+		str_desc = "Rows of three-story brick condominiums with white marble moulding wind along lanes of old asphalt roads with faded markings. Spiked wrought-iron gates protect the lawn of the district’s principal institutions, like the senior center.\nOld New Yonkers is popular with the older citizens of the city, due to it’s incredibly boring, gentrified residential landscape. Modest outdoor malls sells useless shit like candles and soaps, and the elderly population fills up their lumpy, sagging bodies at chain restaurants like Applebee’s and fucking IHOP.\n\nTo the Northeast is New New Yonkers. To the Southeeast is Vagrant's Corner. To the Southwest is Smogsburg. To the East is Little Chernobyl. To the Northwest is Brawlden.",
```
Comment (magicksRampage) [inline ewcfg.py]: "Southeeast" should be "Southeast"

**Comment by munchypizza** [comment]
Fixed all the "it's" and other typo fixes

**Pull request merged**